pkg/cmd/pipelinerun/logs_test.go

```diff
@@ -277,7 +302,7 @@ func TestPipelinerunLogs(t *testing.T) {
 			),
 		),
 	}
-
+	fmt.Println("PipelineRuns are -", prs) // Mine
```

Comment:
This looks like it was from debugging. Please remove it.

Reply:
I am sorry about that. I have made the correction.